NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -505,7 +505,7 @@ class HTauTauNtuplizer : public edm::EDAnalyzer {
   std::vector<Int_t> _particleType;//0=muon, 1=e, 2=tau
   std::vector<Float_t> _combreliso;
   std::vector<Float_t> _combreliso03;
-  std::vector<Float_t> _discriminator;//BDT for ele, discriminator for tau, 
+  //std::vector<Float_t> _discriminator;//BDT for ele, discriminator for tau, 
```

Comment:
it's ok to delete this for electrons and muons (we don't use it), but did you also check for taus?

Reply:
ok I checked in KLUB and it is not used anymore, so it is ok to comment it